charts/elastic-cloud/templates/secrets.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: aws
-  namespace: {{ .Release.Namespace }}
+  name: elastic-cloud-aws-credentials
```

This name should be generated via the helm installation name.

 ```suggestion
  name: {{ template "elastic-cloud.fullname" . }}-aws-credentials
```

By default, it'd evaluate to ` elastic-cloud-aws-credentials`.


And can be overrideen via `.Values.fullnameOverride` (see [_helpers.tpl](https://github.com/rhoai/helm-charts/blob/master/charts/elastic-cloud/templates/_helpers.tpl#L14))